### PR TITLE
uninstall only S3 portion of openshift-glusterfs

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
when openshift_storage_glusterfs_s3_only_uninstall is True.

Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

**Comment by SaravanaStorageNetwork** [comment]
> you should probably create a new uninstall_s3.yml playbook in playbooks/openshift-glusterfs/.

Sure. It makes sense..will update

**Comment by SaravanaStorageNetwork** [comment]
@jarrpa  PTAL - updated as per your comment - Now directly invoking the playbook "playbooks/openshift-glusterfs/uninstall_s3.yml" removes all S3 resources created

File: playbooks/openshift-glusterfs/private/uninstall_s3.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: Uninstall GlusterFS
```
Comment (jarrpa) [inline playbooks/openshift-glusterfs/private/uninstall_s3.yml]: This should be `Uninstall gluster-s3`

**Comment by vrutkovs** [comment]
Master branch is closed! A major refactor is ongoing in devel-40. Changes for 3.x should be made directly to the latest release branch they're relevant to and backported from there.

**Pull request closed without merging**